mne/preprocessing/flat.py

```diff
@@ -33,6 +33,15 @@ def annotate_flat(raw, bad_percent=5., min_duration=0.005, picks=None,
         time samples with exactly the same value are not totally uncommon.
         Defaults to 0.005 (5 ms).
     %(picks_good_data)s
+    flatness : dict | float
```

Comment:
this param is called flat in https://mne.tools/stable/generated/mne.Epochs.html?highlight=epochs#mne.Epochs

shall we be consistent here and call it flat here too?